src/network/parallelconnection_lua.inl

```diff
@@ -101,6 +101,29 @@ int setPassword(lua_State* L) {
     }
 }
 
+int setHostPassword(lua_State* L) {
+    bool isFunction = (lua_isfunction(L, -1) != 0);
```

Comment:
You can check the number of arguments first with:
```
    int nArguments = lua_gettop(L);
    SCRIPT_CHECK_ARGUMENTS("loadScene", L, 1, nArguments);
```

`SCRIPT_CHECK_ARGUMENTS` is defined in `openspace/scripting/script_helper.h`